### PR TITLE
Incubation Exit and Graduation Proposal for Cacti

### PR DESCRIPTION
**Pull request by VRamakrishna**
Added a new file containing details about how Cacti meets the Hyperledger graduation requirements. See [here](https://github.com/hyperledger/toc/issues/164) for the corresponding issue (with a PPT) opened in the TOC repository.

### CODE REVIEW

File: HIPs/cacti-graduation.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+- As we are embedded in this ecosystem, we will ensure that Cacti code conforms to emerging standards created through community consensus, increasing Cacti’s visibility and value in the community
+
+### Graduation Benefits for Community
+
+- Code is already suitable for production usage, to integrate and couple DLT systems in different modes
+  * Proven by real world usage
+- Graduated status will give enterprises and DLT providers looking for generic interoperation tools
+  * A canonical location to look for those tools or to co-develop or contribute features
+  * More confidence in the maturity of the Cacti code base
+- New contributors will be more enthusiastic about not using as well as contributing (we have received communication of interest)
```
Comment (tkuhrt) [inline HIPs/cacti-graduation.md]:
```suggestion
- New contributors will be more enthusiastic about using as well as contributing (we have received communication of interest)
```

**Comment by tkuhrt** [comment]
**Approved at September 21, 2023 TOC meeting**

- Proposed - Peter
- Seconded - Arun

Unanimous, 8/10 active TOC members:

- Tracy - For
- Arun - For
- Rama - For
- Bobbi - For
- StephenC - For
- Jim - For
- Peter - For
- Marcus - For

**Review by tkuhrt — APPROVED** [review]

File: HIPs/cacti-graduation.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  * Configured as GitHub Actions, mandatory for PRs to pass
+- Currently we don’t have a unified test coverage report, but we are planning to generate a table with coverage for each package listed
+
+### User Documentation
+
+- Usage and tutorial documentation exist for the Cactus and Weaver packages respectively
+  * Instructions to build and deploy (through a local web server) exist
+  * Contain details of setups, prerequisites, installations, and evaluations (end-to-end sample runs)
+- An integrated Cacti documentation website with tutorials and reference information is in the works
+  * Built according to Tracy’s mkdocs template
```
Comment (arsulegai) [inline HIPs/cacti-graduation.md:55]: [nitpick] It will help if mkdocs template link is referenced here.

File: HIPs/cacti-graduation.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+  * Cactus whitepaper
+  * Weaver RFCs
+
+### Alignment
+
+- The goals, mission, and use case scenarios addressed by Cacti are amply documented in README files as well as in the whitepaper, RFCs, and tutorial docs
+- As a project [designed to promote and enable interoperability](https://github.com/hyperledger/cacti#scope-of-project), Cacti conforms to, fulfils, and augments, the Hyperledger architecture and mission
+  * Not just compatible with, but also enhances the capabilities of Hyperledger blockchain technologies like Fabric, Besu, Sawtooth, and Iroha, by making cross-network transactions possible and seamless, and without impacting the networks’ internal operations
+  * Also supports non-Hyperledger DLTs like R3 Corda (and Polkadot), allowing Hyperledger blockchains to be interoperable with non-Hyperledger systems without internal modifications
+  * Will use Caliper for performance benchmarking and ensure interoperability with Firefly in the future
```
Comment (arsulegai) [inline HIPs/cacti-graduation.md:67]: +1

**Review by arsulegai — APPROVED** [review]

**Pull request merged**